guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md

```diff
@@ -0,0 +1,141 @@
+# Guardrail example: Groundedness check
+
+One way of mitigating hallucination by an agent, is to check for each agent's response if it is grounded in the content. One way of doing this is by using Azure Content Safety's Groundedness API.
+
+## Create an Azure Content Safety instance
+
+In the [Azure Portal](https://portal.azure.com), type **Azure Content Safety** in the search bar and create an new instance if needed.
+
+Please use **East US** as the region for deployment. As of today when writing this lab content, this is the only region which has all (preview) features available. You can review the [regional availability for ACS here](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview#region-availability).
```

Comment:
Just checking, is this still true that `East US` is on the only region?

Reply:
Yes, according to this documentation: https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview#region-availability